www/docs/en/dev/guide/appdev/hooks/index.md

```diff
@@ -22,61 +22,191 @@ title: Hooks Guide
 
 # Hooks Guide
 
+## Introduction
+
 Cordova Hooks represent special scripts which could be added by application and
 plugin developers or even by your own build system  to customize cordova commands.
-Hook scripts could be defined by adding them to the special predefined folder 
-(`/hooks`) or via configuration files (`config.xml` and `plugin.xml`) and run
-serially in the following order:
-
-* Application hooks from `/hooks`;
-* Application hooks from `config.xml`;
-* Plugin hooks from `plugins/.../plugin.xml`.
 
-__Note__: `/hooks` directory is considered deprecated in favor of the hook elements
-in config.xml and plugin.xml.
-
-## Supported hook types
-The following hook types are supported:
-
-    after_build
-    after_compile
-    after_clean
-    after_docs
-    after_emulate
-    after_platform_add
-    after_platform_rm
-    after_platform_ls
-    after_plugin_add
-    after_plugin_ls
-    after_plugin_rm
-    after_plugin_search
-    after_plugin_install // Plugin hooks in plugin.xml are executed for a plugin being installed only
-    after_prepare
-    after_run
-    after_serve
-    before_build
-    before_clean
-    before_compile
-    before_docs
-    before_emulate
-    before_platform_add
-    before_platform_rm/
-    before_platform_ls
-    before_plugin_add
-    before_plugin_ls
-    before_plugin_rm
-    before_plugin_search/
-    before_plugin_install // Plugin hooks in plugin.xml are executed for a plugin being installed only
-    before_plugin_uninstall // Plugin hooks in plugin.xml are executed for a plugin being uninstalled only
-    before_prepare
-    before_run
-    before_serve
-    pre_package // Windows and Windows Phone only
+There may be special activities that you would like to perform before invoking 
```

Comment:
Consider saying 'Cordova hooks allow you to perform special activities around cordova commands'.